crema/parsers/txt.py

```diff
@@ -0,0 +1,128 @@
+"""A parser for generic delmited text files."""
+import logging
+
+import pandas as pd
+from ..dataset import PsmDataset
+from ..utils import listify
+
+LOGGER = logging.getLogger(__name__)
+
+
+def read_txt(
+    txt_files,
+    target_column,
+    spectrum_columns,
+    score_columns,
+    peptide_column,
+    sep="\t",
+    copy_data=True,
+):
+    """Read peptide-spectrum matches (PSMs) from delimited text files.
+
+    Parameters
+    ----------
+    txt_files : str, panda.DataFrame, or tuple of str
+        One or more collection of PSMs in a tabular text format.
+    target_column : str
+        The column that indicates whether a PSM is a target or a decoy.
+    spectrum_columns : str or tuple of str
+        One or more columns that together define a unique mass spectrum.
+    score_columns : str or tuple of str, optional
+        One or more columns that indicate scores by which crema can rank PSMs.
+    peptide_column : str
+        The column that defines a unique peptide. Modifications should be
+        indicated either in square brackets :code:`[]` or parentheses
+        :code:`()`. The exact modification format within these entities does
+        not matter, so long as it is consistent.
+    copy_data : bool, optional
```

Comment:
Add docstring for "sep" parameter? You might have intentionally skipped this since you gave it a default value though, just thought I'd check.